pkg/plugins/executor/template_executor_plugin.go

```diff
@@ -0,0 +1,48 @@
+package executor
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+)
+
+// swagger:parameters executeTemplate
+type ExecuteTemplateRequest struct {
+	// in: body
+	// Required: true
+	Body ExecuteTemplateArgs
+}
+
+type ExecuteTemplateArgs struct {
+	// Required: true
+	Workflow *Workflow `json:"workflow"`
+	// Required: true
+	Template *wfv1.Template `json:"template"`
+}
+
+// swagger:response executeTemplate
+type ExecuteTemplateResponse struct {
+	// in: body
+	Body ExecuteTemplateReply
+}
+
+type ExecuteTemplateReply struct {
+	Node    *wfv1.NodeResult `json:"node,omitempty"`
+	Requeue *metav1.Duration `json:"requeue,omitempty"`
```

Comment:
Should we document the requeue part of the response in executor_plugins.md?

Reply:
done